src/rates.ts

```diff
@@ -183,6 +183,17 @@ export const getExchangeRates = async (
     const documents: DbDoc[] = await getFromDb(localDb, docs)
 
     const out = await getRatesFromProviders({ data, documents }, edgeAssetDoc)
+
+    // Save USD rates to Redis
+    for (const doc of out.documents) {
+      const newRates = Object.keys(doc)
+        .filter(key => key.includes('iso:USD'))
+        .map(pair => [pair, doc[pair]])
+        .flat()
```

Comment:
I've been wanting to use this in the GUI for sooooo long. But I was afraid that updating the target would break it.
Now I'm definitely going to try ;)